spec/unit/lib/cadence/workflow/serializer/query_answer_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'cadence/workflow/serializer/query_failure'
+require 'cadence/workflow/query_result'
+require 'cadence/workflow/serializer/query_answer'
+
+describe Cadence::Workflow::Serializer::QueryAnswer do
+  class TestDeserializer
+  end
+
+  describe 'to_thrift' do
+    let(:query_result) { Cadence::Workflow::QueryResult.answer(42) }
+    it 'produces a thrift object' do
+      result = described_class.new(query_result).to_thrift
+
+      expect(result).to be_a(CadenceThrift::WorkflowQueryResult)
+      expect(result.resultType).to eq(CadenceThrift::QueryResultType::ANSWERED
+                                   )
+      expect(result.answer).to eq("42")
```

Comment:
I tried to call `JSON.serialize(42)`, but I keep getting `NoMethodError: undefined method serialize for JSON:Module`.

Reply:
@djung335 try `Cadence::JSON.serialize`